scripts/release.sh

```
@@ -0,0 +1,67 @@
+
+CMD=$1
+SQL=$2
+PORT=$3
+
+shift
+shift
+shift
+
+prepare $SQL
+localtonet $PORT
+boot $CMD $@
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)